scripts/firezone-client-gui-install.sh

```diff
@@ -3,22 +3,30 @@
 #
 # The `./` is necessary
 #
-# This script should be idempotent
+# This script should be idempotent, so it can be used for upgrades, too.
 set -euox pipefail
 
 # `apt-get` needs either a leading `./` or `/` to recognize a local file path
 DEB_PATH=$(realpath "$1")
+GROUP_NAME="firezone-client"
+SERVICE_NAME="firezone-client-ipc"
 
 echo "Installing Firezone..."
 sudo apt-get install --yes "$DEB_PATH"
 
-echo "Adding your user to the firezone-client group..."
+echo "Adding your user to the $GROUP_NAME group..."
 # Creates the system group `firezone-client`
 sudo systemd-sysusers
```

Comment:
apt doesn't run this for us? Boo. On Arch it does.

Reply:
We might be able to do it with a post-install script https://github.com/firezone/firezone/issues/4987
But the Tauri deb bundler hides a lot of customization, at least in 1.x

Reply:
> We might be able to do it with a post-install script #4987

I am surprised it needs custom scripting at all and this isn't done by a hook if the file is part of the deb and placed in the correct directory.

Reply:
Yeah it's odd but based on what I'm seeing here they've just been doing it with post-inst scripts this whole time? https://unix.stackexchange.com/a/147123 https://wiki.debian.org/AccountHandlingInMaintainerScripts

It's for user accounts but I assume groups are about the same.

Reply:
#5011 fixed some of this